### DISCUSSION TITLE
[V5] How to change Default Timezone?

### DISCUSSION THREAD

**Discussion by theashywolf** [adonisjs/core#2055] [General]
Im stuck with this
I need to record time for where I service (Tokyo, GST+9)
also I tried to change luxon's config but seems not effective.
what should I do?

**Comment by theashywolf** [comment]
Solved: 
there is timezone option in configs/database.ts

**Reply by khaeroen** [reply]
@theashywolf .. may i see your timezone option in database.ts ? I search in config/database.ts but didn't find any clue where timezone option..

**Reply by McSneaky** [reply]
Also setting `TZ` env variable might help. This way it sets whole NodeJS timezone (`TZ=utf` for example)
